metrics-api/src/main/java/io/enmasse/metrics/api/PrometheusMetricsFormatter.java

```diff
@@ -4,19 +4,37 @@
  */
 package io.enmasse.metrics.api;
 
+import java.io.IOException;
 import java.util.List;
+import java.util.Properties;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 
 public class PrometheusMetricsFormatter implements MetricsFormatter {
+    private static final Logger log = LoggerFactory.getLogger(PrometheusMetricsFormatter.class.getName());
     public String format(List<Metric> metricList) {
         StringBuilder sb = new StringBuilder();
+        String appPrefix = "";
+
+        try {
+            java.io.InputStream inputStream = this.getClass().getResourceAsStream("/pom.properties");
```

Comment:
getResourceAsStream returns null if the file is not found.  You should throw an exception if null is returned.